src/main/target/common_post.h

```diff
@@ -195,6 +195,13 @@
 #undef USE_ADC_INTERNAL
 #endif
 
+#define USE_FLASH // blckmn - can we make this a build option? -- todo fix the unified targets to reference flash groups
```

Comment:
@blckmn we talked about the flash option

Reply:
@haslinghuis Flash option added :)

Reply:
Nice. Removed this line.